### PR TITLE
Fixing docs

### PR DESCRIPTION
**Pull request by elamaran11**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
 ## AWS Observability Accelerator
 
-[![pre-commit](https://github.com/aws-observability/aws-observability-accelerator/actions/workflows/pre-commit.yaml/badge.svg?branch=main)](https://github.com/aws-observability/aws-observability-accelerator/actions/workflows/pre-commit.yaml)
```
Comment (bonclay7) [inline README.md:3]: Why removing pre-commit badge?

File: README.md [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
-
 Welcome to the AWS Observability Accelerator!
 
 The AWS Observability Accelerator help you set up observability for your AWS environments with AWS-managed observability services such as Amazon Managed Service for Prometheus, Amazon Managed Grafana, AWS Distro for OpenTelemetry (ADOT) and Amazon CloudWatch.
 
-This repository provide shared artifacts (documentation, dashboards, alerting rules) for the [Terraform](https://github.com/aws-observability/terraform-aws-observability-accelerator) project and serves as a GitOps source repository.
+This repository provide shared artifacts (documentation, dashboards, alerting rules) for the [Terraform](https://github.com/aws-observability/terraform-aws-observability-accelerator) project and [CDK](https://github.com/aws-observability/cdk-aws-observability-accelerator) project. This repository also serves as a GitOps source repository.
```
Comment (bonclay7) [inline README.md:7]: Can we say `for the Terraform and CDK projects` instead of repeating the word project?

File: README.md [hunk 3]
```diff
@@ -1,14 +1,12 @@
...
 
-Checkout the [project documentation](https://aws-observability.github.io/terraform-aws-observability-accelerator/) for more informations.
+Checkout the [project documentation for Terraform](https://aws-observability.github.io/terraform-aws-observability-accelerator/) and [project documentation for CDK](https://aws-observability.github.io/cdk-aws-observability-accelerator/) for more information.
```
Comment (bonclay7) [inline README.md:9]: same comment for "project documentation"

File: README.md (hunk 2 above)
Comment (elamaran11) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (elamaran11) [reply README.md]: fixed

**Comment by elamaran11** [comment]
@bonclay7 Fixed the issues pointed.

**Review by bonclay7 — APPROVED** [review]
Thank you

**Pull request merged**